### PR TITLE
[type feature] add shenyu-agent-plugin-tracing-zipkin module.

### PR DESCRIPTION
**Pull request by midnight2104**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <parent>
+        <groupId>org.apache.shenyu</groupId>
+        <artifactId>shenyu-agent-plugin-tracing</artifactId>
+        <version>2.4.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>shenyu-agent-plugin-tracing-zipkin</artifactId>
+
+    <properties>
```
Comment (yu199195) [inline shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/pom.xml:31]: remove this

File: shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/src/main/java/org/apache/shenyu/agent/plugin/tracing/zipkin/constant/ZipkinConstants.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.agent.plugin.tracing.zipkin.constant;
+
+/**
+ * The type Zipkin constants.
+ */
+public final class ZipkinConstants {
```
Comment (yu199195) [inline shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/src/main/java/org/apache/shenyu/agent/plugin/tracing/zipkin/constant/ZipkinConstants.java:23]: can used common？

File: shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/pom.xml (hunk 1 above)
Comment (midnight2104) [reply shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/pom.xml]: done, remove it

File: shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/src/main/java/org/apache/shenyu/agent/plugin/tracing/zipkin/constant/ZipkinConstants.java (hunk 2 above)
Comment (midnight2104) [reply shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-tracing/shenyu-agent-plugin-tracing-zipkin/src/main/java/org/apache/shenyu/agent/plugin/tracing/zipkin/constant/ZipkinConstants.java]: I have deleted it and will extract it when I need it.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**